### PR TITLE
chore: added commitizen

### PR DESCRIPTION
**Pull request by Berkmann18**
**What**:
I added `commitizen` with the angular adapter and the commitizen badge (so contributors know what to expect using when they commit).

**Why**:
I thought it would be good to have a non-restrictive (ie. contributors/members can still use `git commit`) commit process which ensures that commits follow the angular commit convention while ensuring that the commits are structured properly (which is important when a changelog generation process is in place).

**How**:
- Added `git-cz` and `cz-conventional-changelog` to the `devDeps` including a `commit` script (although it can be run using `git cz` (which worked for me) if the contributor has installed it globally).
- Copy-pasted the badge from [here](https://github.com/commitizen/cz-cli).

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

**Comment by jakebolam** [comment]
@Berkmann18 commitizen change looks good. Looks like this PR has the #142 work too

**Comment by Berkmann18** [comment]
Yeah, whoops should have branched it out off `master`!

**Comment by jakebolam** [comment]
Sweet, yeah will approve when the other one is in. (or this is re-based)

**Comment by Berkmann18** [comment]
Okay, I've rebased this as it was apparently out of date with `master`.
Once, this PR is merged, I'll get it done for the 3 other repos.

File: README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ Where `username` is the user's GitHub or Gitlab username, and `contribution` is
 - tutorial: [✅](# 'Tutorials')
 - userTesting: [📓](# 'User Testing')
 - video: [📹](# 'Videos')
+- maintenance [👷](# 'Maintenance')
```
Comment (sinchang) [inline README.md]: this change has in pr https://github.com/all-contributors/all-contributors-cli/pull/142

File: src/util/contribution-types.js [hunk 2]
```diff
@@ -99,6 +99,10 @@ const defaultTypes = function(repoType) {
       symbol: '📹',
       description: 'Videos',
     },
+    maintenance: {
```
Comment (sinchang) [inline src/util/contribution-types.js]: this change has in pr https://github.com/all-contributors/all-contributors-cli/pull/142

Comment (Berkmann18) [reply src/util/contribution-types.js]: I've removed that and the line on the README file.

**Pull request merged**